ios/src/TGMapViewController.h

```diff
@@ -22,13 +42,34 @@
 - (void)recognizer:(UIGestureRecognizer *)recognizer didRecognizeShoveGesture:(CGPoint)location;
 @end
 
+NS_ASSUME_NONNULL_END
+
 struct TileID;
 @interface TGMapViewController : GLKViewController <UIGestureRecognizerDelegate>
 
-@property (assign, nonatomic) Tangram::Map* map;
+@property (assign, nonatomic, nullable) Tangram::Map* map;
 @property (assign, nonatomic) BOOL continuous;
-@property (weak, nonatomic) id<TangramGestureRecognizerDelegate> gestureDelegate;
+@property (weak, nonatomic, nullable) id<TangramGestureRecognizerDelegate> gestureDelegate;
+
+// The following are computed property. They return sensible defaults when the above .map property is nil
+@property (assign, nonatomic) TangramCameraType cameraType;
+@property (assign, nonatomic) TangramGeoPoint position;
+@property (assign, nonatomic) float zoom;
+@property (assign, nonatomic) float rotation;
+@property (assign, nonatomic) float cameraTilt;
```

Comment:
`tilt` seems more in line with the other property names than `cameraTilt`. The [MapKit camera](https://developer.apple.com/library/ios/documentation/MapKit/Reference/MKMapCamera_class/index.html#//apple_ref/occ/instp/MKMapCamera/pitch) uses "pitch" for this, and "bearing" for rotation (sort of) but we don't have to match it exactly.